### ISSUE TITLE
Add support for font

### ISSUE THREAD

**Issue by Splaxi** [XjSv/environment-marker#19]
Awesome little extension that you have created. It resembles the other one, but have a few enhancements that I like.

I think you should consider adding support for the user to change font. It would increase the usability a lot. It just needs to be a general font, and not per url, like the coloring.

**Comment by Splaxi** [comment]
An even more awesome feature, along the font selection - would be font size. But the size should be per url, because that could mitigate the need for #20, because the user could then simply adjust the font size directly.

The size selector you have now, should be kept - but only handle the size of the colored area and give the normal font size used with that.

**Comment by XjSv** [comment]
Thank you! I assume you are a Chrome user. I did see the other Chrome extension that is similar to this one but unfortunately I am a Firefox user and when I developed this there was nothing like it for Firefox. I believe this is still the case for Firefox. As for Chrome, I did not mean to compete with the other extension, I just simply standardized some code so it universally works for all browsers that support WebExtensions. I am planning on submitting it for Safari also.

As for the font selector and size I think they are great features to add. Ill be working on them and including them in the upcoming v2.5 release. Thanks for suggesting this.
